containerservice/managedcluster_test.go

```diff
@@ -0,0 +1,101 @@
+package containerservice
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"os"
+	"strconv"
+	"testing"
+
+	"github.com/Azure-Samples/azure-sdk-for-go-samples/helpers"
+	"github.com/Azure-Samples/azure-sdk-for-go-samples/resources"
+)
+
+var (
+	resourceName     string
+	username         = "azureuser"
+	sshPublicKeyPath = os.Getenv("HOME") + "/.ssh/id_rsa.pub"
+	clientID         string
+	clientSecret     string
+	agentPoolCount   int32
+)
+
+func TestMain(m *testing.M) {
+	err := parseArgs()
+	if err != nil {
+		log.Fatalln("failed to parse args")
+	}
+
+	ctx := context.Background()
+	defer resources.Cleanup(ctx)
+
+	_, err = resources.CreateGroup(ctx, helpers.ResourceGroupName())
+	if err != nil {
+		helpers.PrintAndLog(err.Error())
+	}
+	helpers.PrintAndLog(fmt.Sprintf("resource group created on location: %s", helpers.Location()))
+
+	os.Exit(m.Run())
+}
+
+func parseArgs() error {
+	err := helpers.ParseArgs()
+	if err != nil {
+		return fmt.Errorf("cannot parse args: %v", err)
+	}
+
+	resourceName = os.Getenv("AZ_RESOURCE_GROUP_NAME")
```

Comment:
Resource group?

Reply:
I kept the resource name (cluster name) same as resource group name. If preferred, I can make it configurable. Lmk what you think.

Reply:
Sorry, I mean the environment variable name, maybe `AZ_RESOURCE_NAME` instead of `AZ_RESOURCE_GROUP_NAME`?

Reply:
Sounds good, changed to `AZ_AKS_NAME`